plugin/pulpcore/plugin/stages/declarative_version.py

```diff
@@ -0,0 +1,146 @@
+import asyncio
+from gettext import gettext as _
+
+from pulpcore.plugin.models import RepositoryVersion
+from pulpcore.plugin.tasking import WorkingDirectory
+
+from .api import create_pipeline, EndStage
+from .artifact_stages import ArtifactDownloader, ArtifactSaver, QueryExistingArtifacts
+from .association_stages import ContentUnitAssociation, ContentUnitUnassociation
+from .content_unit_stages import ContentUnitSaver, QueryExistingContentUnits
+
+
+class FirstStage:
```

Comment:
Why is `FirstStage` still needed?

Reply:
It's not, but I was too tired to remove it last night. It'll be removed in next push.